MultiTable/src/SampleUtils.js

```diff
@@ -91,8 +91,8 @@ module.exports = class SampleUtils {
 
   static isUnderRootSubject(sample, rootSubject) {
     return sample.name.toLowerCase()
-      .indexOf(rootSubject.toLowerCase()) === 0;
-  }
+      .startsWith(rootSubject.toLowerCase()) === true;
```

Comment:
I agree with this change, but can you please file a new bug to lock down the hole in the core Refocus app where we allow you to create a perspective with an empty string for the rootSubject? It's a required field in swagger, but we should probably do some validation on the string to confirm that there is a subject with that actual absolutePath.

Reply:
code change here requires update to lens.json and the zip you build under /dist

Reply:
Yes, will create the story. Also, this change  returns true for  "someString".startsWith(""). I changed it because the best-case and worst case time complexity of startsWith is O(n), vs the indexOf function which has a worst case complexity of O(n^2).  I initially had something like 
`rootSubject.length > 0  && sample.name.toLowerCase()
      .startsWith(rootSubject.toLowerCase()) === true` but removed the `rootSubject.length > 0`. Should I add it back ?

Reply:
Yeah, I think it's safer to add back `rootSubject.length > 0` check